src/CloudEvents/Serializer/Exception/UnsupportedVersionException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace CloudEvents\Serializer\Exception;
+
+/**
+ * Thrown when the serializer does not support the spec version of the CloudEvent instance.
+ */
+class UnsupportedVersionException extends SerializationException
```

Comment:
This comment can be removed by renaming the class to be more clear.

```suggestion
class UnsupportedSpecVersionException extends SerializationException
```